docs/docs/reference/built-in-components/gatsby-head.md

```diff
@@ -57,6 +57,28 @@ export default Page
 export { Head } from "../another/location"
 ```
 
+### Editing `<html>` and `<body>`
+
+> Support for the editing `<html>` and `<body>` added in `gatsby@5.5.0`.
+
+You can set `<html>` and `<body>` attributes:
+
+```jsx
+export function Head() {
+  return (
+    <>
+      <!-- highlight-start -->
+      <html lang="en">
+      <body className="my-body-class">
+      <!-- highlight-end -->
+      <title>Hello World</title>
+    </>
+  )
+}
+```
+
+Gatsby will use provided attributes and inject them into resulting html.
```

Comment:
Gatsby will pick up attributes on these tags and inject them into resulting html.